src/DotNetWorker.OpenTelemetry/release_notes.md

```diff
@@ -1,5 +1,5 @@
 ## What's Changed
 
-### Microsoft.Azure.Functions.Worker.OpenTelemetry 1.0.0 (Preview 2)
+### Microsoft.Azure.Functions.Worker.OpenTelemetry 1.1.0
 
-- `UseFunctionsWorkerDefaults` signature now uses `IOpenTelemetryBuilder` (from `OpenTelemetryBuilder`) (#2839)
\ No newline at end of file
+- Updated OpenTelemetry NuGet packages and aligned implementation with Semantic Conventions v1.37. (#3195)
```

Comment:
Since we are GAing this package , should we include entries from other preview versions here? the full change logs?